Makefile.link

```diff
@@ -0,0 +1,23 @@
+#############################################################################
```

Comment:
this file is outdated and does not include the internal_communication please update your build repo and copy the new makefiles from the template-arduino